package.json

```diff
@@ -10,16 +10,17 @@
   "scripts": {
     "build": "webpack --config webpack.config.js --mode production",
     "build-dev": "webpack --config webpack.config.js --watch --mode development",
-    "seed": "node server/database/seed.js",
-    "server-dev": "nodemon server/",
-    "start": "node server/",
+    "seed": "node -r dotenv/config server/database/seed.js",
+    "server-dev": "nodemon -r dotenv/config server/",
+    "start": "node -r dotenv/config server/",
```

Comment:
I would love to know what you are doing here.

Reply:
I'm pre-loading the environment variables... let's talk about it after our call with Robin. :)